build/yamls/base/agent.yml

```diff
@@ -87,6 +87,9 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: spec.nodeName
+            # For using noEncap or hybrid traffic mode without AntreaProxy
+            - name: TRAFFIC_DIRECT_ROUTING
```

Comment:
I don't understand this name. I would think that something like `ALLOW_NO_ENCAP_WITHOUT_ANTREA_PROXY` would be more meaningful.

Reply:
Agree, the variable name has been modified.